app/dashboard/templates/addinterest.html

```diff
@@ -124,6 +124,18 @@ <h5 class="text-center font-title">{% trans "Submit a Plan" %}</h5>
                 <input type="url" class="form-control" name="work_url" id="work_url" placeholder="https://github.com/gitcoinco/web" required>
               </div>
 
+              <div class="form-group mt-4">
+                <label class="font-weight-semibold" for="videodemo_url">Select video provider</label>
+                <select name="videodemo-provider">
+                  <option value=""><i class="fa fa-youtube"></i> Youtube</option>
+                  <option value=""><i class="fa fa-vimeo">Vimeo</option>
+                  <option value=""><i class="fa fa-loom">Loom</option>
+                  <option value=""><i class="fa fa-video">HTML5 video</option>
+                </select>>
+                <label class="font-weight-semibold" for="videodemo-url">Link to Video Demo</label>
+                <input type="url" class="form-control" name="work_url" id="work_url" placeholder="https://github.com/gitcoinco/web" required>
+              </div>
+
```

Comment:
if someone pick html5 are we going to host the video?

Reply:
No, this is URL only. This could be hosted for example in skynet. I recently see some hackathon demos hosted in skynet.